Doc/howto/perf_profiling.rst

```diff
@@ -147,19 +148,29 @@ Instead, if we run the same experiment with perf support activated we get:
 
 
 
-Enabling perf profiling mode
-----------------------------
+How to enable ``perf`` profiling support
+----------------------------------------
 
-There are two main ways to activate the perf profiling mode. If you want it to be
-active since the start of the Python interpreter, you can use the `-Xperf` option:
+There are three ways to enable ``perf`` profiling support:
+
+* using the :option:`-Xperf <-X>` command-line option
```

Comment:
IMO ``-X perf`` is more readable than ``-Xperf``, I suggest replacing it in the doc like this:

```suggestion
* using the :option:`-X perf <-X>` command-line option
```